phonebuddies/PhoneNumberParser.py

```diff
@@ -0,0 +1,52 @@
+import re
+import phonenumbers
+class PhoneNumberParser:
+    @staticmethod
+    def parse_to_valid_whatsapp(phone_number):
```

Comment:
`phone_number` should perhaps be `unparsed_number` or something unless you plan on changing it

Reply:
Idea for algorithm:
1. Remove whitespace and special chars
1. Give it to `phonenumbers` and see whether it's valid. If valid, return that number.
1. If invalid, start making assumptions and document those. => Check the error (is it missing a region? can we reparse assuming a US/GB/Afghan region?)